WebMvc/Models/CartModels/CartItem.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace WebMvc.Models.CartModels
+{
+    public class CartItem
+    {
+        public string Id { get; set; }
+        public string ProductId { get; set; }
```

Comment:
```suggestion
        public string EventId { get; set; }
```
just so we have consistent names across our services